images/worker/entrypoint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -e
+
+# Link Frappe's node_modules/ to make Website Theme work
+if test -d /home/frappe/frappe-bench/sites/assets/frappe/node_modules; then
```

Comment:
What if we make broken symlink in Dockerfile instead and when assets volume is there it would be valid

Reply:
does it work as expected?

Reply:
Don’t know. Can you test?

Reply:
okay.

Reply:
this worked. I reset the changes in PR, no `--chown` and `entrypoint.sh` used. Then I made following changes to frappe_builder

```
FROM build_deps as frappe_builder

ARG FRAPPE_VERSION
RUN --mount=type=cache,target=/home/frappe/.cache/pip \
    git clone --depth 1 -b ${FRAPPE_VERSION} https://github.com/frappe/frappe apps/frappe \
    && env/bin/pip install -e apps/frappe \
    && env/bin/pip install -U gevent \
    && rm -r apps/frappe/.git \
    && mkdir -p /home/frappe/frappe-bench/sites/assets/frappe/node_modules \
    && ln -s /home/frappe/frappe-bench/sites/assets/frappe/node_modules /home/frappe/frappe-bench/apps/frappe/node_modules
```

I had to create a directory and make a symlink. That became active in container. I could save website theme.

Reply:
That is great!

Reply:
I'll close this PR and create a fresh one with only this change.